scripts/linters/test_files/invalid_request.py

```diff
@@ -21,9 +21,9 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
-import urllib2
+from core import python_utils
 
-import python_utils
+import urllib2
```

Comment:
Shouldn't the stdlib take priority over Oppia-specific stuff?

Reply:
Yep, probably not caught because this is just a testing file.